Katello/katello
# Fixes #31929 - Update Angular to 1.8.2

> we need to update to "angular": ">=1.8.0" and "bootstrap-select": ">=1.13.6"

### To test
Run `npm install` in foreman directory 
`cd ../katello`
verify that `npm ls` returns the following
```
[vagrant@centos7-katello-devel katello]$ npm ls angular
katello@1.0.0 /home/vagrant/katello
└── angular@1.8.2 

[vagrant@centos7-katello-devel katello]$ npm ls bootstrap-select
katello@1.0.0 /home/vagrant/katello
└── bootstrap-select@1.13.6 
```
also note version `1.8.2` of angular inside Bastion: https://github.com/Katello/katello/pull/9184/files#diff-5a25bf98df9f2be0b7eb7c0881ece9b12ca0bc3a11ad7383896dade7065efd90R2831
verify that bastion pages load correctly with no errors
